libr/core/cmd_debug.c

```diff
@@ -1571,6 +1572,34 @@ static int r_debug_heap(RCore *core, const char *input) {
 	return true;
 }
 
+static bool get_bin_info(RCore *core, const char *file, ut64 baseaddr, int mode, bool symbols_only, const char *symname) {
+	int fd;
+	if ((fd = r_io_fd_open (core->io, file, R_PERM_R, 0)) != -1) {
```

Comment:
just do early return with
```C
if ((fd = r_io_fd_open (....) == -1) {
   return false;
}
```
so all the rest of the code doesn't need to be indented.